framework/include/utils/InputParameters.h

```diff
@@ -313,6 +339,52 @@ class InputParameters : public libMesh::Parameters
                            const std::string & syntax,
                            const T & value,
                            const std::string & doc_string);
+  template <typename T>
+  void addCommandLineParam(const std::string & name,
+                           const std::string & syntax,
+                           const std::initializer_list<typename T::value_type> & value,
+                           const std::string & doc_string)
+  {
+    addCommandLineParam<T>(name, syntax, T{value}, doc_string);
+  }
+
+  /**
+   * Add a command line parameter with an optional value.
+   *
+   * This is a deprecated option and only remains for two parameters:
+   * "mesh_only" and "recover". There are issues with command line
+   * parameters with optional values because if a value following
+   * one of these is a hit cli parameter, we don't know if we should
+   * apply it to the optional option or as a hit parameter.
+   *
+   * @param name The name of the parameer
+   * @param syntax Space separated list of command-line switch syntax that can set this option
+   * @param value The default value to assign
+   * @param doc_string Documentation.  This will be shown for --help
+   */
+  ///@{
```

Comment:
wouldn't this work?

```suggestion
   *@{
   */
```

Reply:
I dunno. We use `///@{` everywhere. Does it matter?